jobs/rootfs-trigger.jpl

```diff
@@ -117,14 +136,24 @@ node("docker && build-trigger") {
                          params.ROOTFS_ARCH)
         }
 
+        stage("BuildRootConfigs") {
+            listBuildrootConfigs(kci_core, buildroot_configs)
+            print(buildroot_configs)
+        }
+
         stage("Build") {
             def builds = [:]
             def i = 0
 
             for (item in configs) {
                 def config_name = item[0]
                 def arch = item[1]
-                def step_name = "${i} ${config_name} ${arch}"
+                def rootfs_type = "debos"
+
+                if (config_name in buildroot_configs) {
+                   rootfs_type = "buildroot"
+	        }
```

Comment:
Broken indentation

Reply:
May be Github issue? I can see correct indentation locally and also here https://github.com/kernelci/kernelci-jenkins/blob/581debaeffdb1df346726e8391e5bc7350aa2c63/jobs/rootfs-trigger.jpl